nansat/domain.py

```diff
@@ -526,9 +526,9 @@ def _create_extent_dict(extent_str):
                 raise ValueError('<extent_dict> must contains exactly 2 parameters '
                                   '("-te" or "-lle") and ("-ts" or "-tr")')
             key, extent_dict = Domain._add_to_dict(extent_dict, option)
-            if key is 'te' or key is 'lle':
+            if key == 'te' or key == 'lle':
```

Comment:
This change highlights a problem in [this test](https://github.com/nansencenter/nansat/blob/45bcb778e55558035a38c2a1fce39632ea791403/nansat/tests/test_nansat.py#L323):
the arguments to `-te` are given in the wrong order.
Could you change `"-te -180 180 60 90 -ts 500 500"` to `"-te -180 60 180 90 -ts 500 500"`?